lib/benches/parser.rs

```diff
@@ -0,0 +1,49 @@
+#![feature(test)]
+
+extern crate test;
+use test::{black_box, Bencher};
+
+macro_rules! parser {
+	($name: ident, $parser: path, $text: expr) => {
+		#[bench]
+		fn $name(b: &mut Bencher) {
+			let text = $text;
+			b.iter(|| {
+				black_box($parser(black_box(text))).unwrap();
+			});
+		}
+	};
+}
+
+parser!(select_simple, surrealdb::sql::parse, "SELECT * FROM person;");
+parser!(
+	select_complex,
+	surrealdb::sql::parse,
+	"SELECT name, age, country FROM person WHERE hair = \"brown\" AND is_vegetarian;"
```

Comment:
How about using single quotes here so we don't have to escape the quotation marks?

Reply:
Good idea!